objdiff.json

```diff
@@ -26,14 +26,6 @@
 
             "reverse_fn_order": false
         },
-        {
```

Comment:
Why remove AnmVm.obj? This breaks the comparison step.

Reply:
It was unintentional. I had restored the version in my local repository's upstream/master branch, but forgot to sync with the remote beforehand and ended up restoring an older version of the file. Sorry about that.